app/src/main/java/com/trongdeptrai/soundcloud/data/model/Track.java

```diff
@@ -0,0 +1,220 @@
+package com.trongdeptrai.soundcloud.data.model;
+
+import android.annotation.SuppressLint;
+import android.os.Parcel;
+import android.os.Parcelable;
+
+@SuppressLint("ParcelCreator")
```

Comment:
remove
https://stackoverflow.com/questions/49602783/parcelcreator-lint-suppression-not-working